{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -50,6 +50,13 @@ repos:
       - id: djlint-reformat-django
       - id: djlint-django
 
+  - repo: https://github.com/astral-sh/uv-pre-commit
+    # uv version.
+    rev: 0.8.14
+    hooks:
+      - id: uv-export
+        args: ['--frozen', '--no-dev']
+
```

Comment:
```suggestion
```

Reply:
@baipaiha can you open a PR against this repo?